### PR TITLE
Improve tasks doc

### PR DESCRIPTION
**Pull request by wldeh**
Refactor the structure for better readability and enhance the language clarity and grammar.

### CODE REVIEW

File: docs/tasks-overview.md [hunk 1]
```diff
@@ -1,45 +1,62 @@
...
+        register("jvm")
+        register("js")
+    }
+}
+```
+
+## Tasks for the "main" Configuration Profile
+
+- **`benchmark`**:
+    - Runs benchmarks within the "main" profile for all registered targets.
```
Comment (qurbonzoda) [inline docs/tasks-overview.md:40]: Could you please also add that "In our example, `benchmark` runs benchmarks within the "main" profile in both `jvm` and `js` targets."

Comment (wldeh) [reply docs/tasks-overview.md]: Added

File: docs/tasks-overview.md [hunk 2]
```diff
@@ -1,45 +1,62 @@
...
+- **`<targetName><configName>Benchmark`**:
+    - Runs benchmarks within `<configName>` profile in `<targetName>` target.
+    - In our example, `jvmSmokeBenchmark` runs benchmarks within the "smoke" profile in `jvm` target while `jsSmokeBenchmark` runs them in `js` target.
+
+## Other useful tasks
+
+- **`<targetName>BenchmarkJar`**:
+    - Created only when a Kotlin/JVM target is registered for benchmarking.
+    - Produces a self-contained executable JAR in `build/benchmarks/<targetName>/jars/` directory of your project that contains your benchmarks in `<targetName>` target, and all essential JMH infrastructure code.
+    - The JAR file can be run using `java -jar path-to-the.jar` command with relevant options. Run with `-h` to see the available options.
```
Comment (qurbonzoda) [inline docs/tasks-overview.md:61]: It is worth mentioning that the JAR file can be used for running JMH profilers

File: docs/tasks-overview.md [hunk 3]
```diff
@@ -1,45 +1,62 @@
...
+        register("jvm")
+        register("js")
+    }
+}
+```
+
+## Tasks for the "main" Configuration Profile
+
+- **`benchmark`**:
+    - Runs benchmarks within the "main" profile for all registered targets. In our example, `benchmark` runs benchmarks within the "main" profile in both `jvm` and `js` targets.
```
Comment (qurbonzoda) [inline docs/tasks-overview.md:40]:
```suggestion
    - Runs benchmarks within the "main" profile for all registered targets.
    - In our example, `benchmark` runs benchmarks within the "main" profile in both `jvm` and `js` targets.
```

Comment (wldeh) [reply docs/tasks-overview.md]: Fixed

**Pull request merged**